src/wp-includes/user.php

```diff
@@ -3012,11 +3012,56 @@ function retrieve_password( $user_login = null ) {
 	 */
 	$message = apply_filters( 'retrieve_password_message', $message, $key, $user_login, $user_data );
 
+	// Wrap the single notification email arguments in an array to pass them to the retrieve_password_notification_email filter.
+	$defaults = array(
+		'to'      => $user_email,
+		'subject' => $title,
+		'message' => $message,
+		'headers' => '',
+	);
+
+	$data = compact( 'key', 'user_login', 'user_data' );
+
+	/**
+	 * Filter the contents of the reset password notification email sent to the user. Return false to prevent sending the email.
```

Comment:
```suggestion
	 * Filter the contents of the reset password notification email sent to the user.
```

Updating docs after we decided against this.